todo_app/app.py

```diff
@@ -1,15 +1,51 @@
-from flask import Flask
-
+from typing import List
+from flask import Flask,escape,request,render_template,redirect
+from flask.helpers import url_for
+from flask.wrappers import Response
+import requests
 from todo_app.flask_config import Config
+from todo_app.data.session_items import TRELLO_BOARD, get_list, add_card, move_card, del_card
+import os
 
 app = Flask(__name__)
 app.config.from_object(Config)
+viewmodel=get_list()
```

Comment:
Note that here we'll want to switch to using your ViewModel class once you have it!